analysed-packages/alsa-lib/version-1.2.6.1/alsa-lib-1.2.6.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,7685 @@
+Copyright (C) 1996-2015 Free Software Foundation, Inc.
+Copyright (C) 2004-2015 Free Software Foundation, Inc.
+Copyright (C) 2010-2015 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: alsa-lib-1.2.6.1.tar.bz2/alsa-lib-1.2.6.1.tar/alsa-lib-1.2.6.1/depcomp
+SPDXID: SPDXRef-item951669
+FileChecksum: SHA1: d429169aeaa44fd5b17f40defacdb9d6767e389a
+FileChecksum: SHA256: b31b449b11f40156b18d2e2a989d78ca3f89be0b96ca0a160455b3299b82d2e6
+FileChecksum: MD5: 91b287ce6c245893a75bd18dc53fb07a
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception-2.0
+FileCopyrightText: <text> Copyright (C) 1999-2020 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: alsa-lib-1.2.6.1.tar.bz2/alsa-lib-1.2.6.1.tar/alsa-lib-1.2.6.1/configure
+SPDXID: SPDXRef-item951246
+FileChecksum: SHA1: 4036843a6e9e89e749160c79d50c4c745c173bae
+FileChecksum: SHA256: 50c0a743183c401abad834ca318559484aa3b7332b1ed1643f4c62ab1993ba33
+FileChecksum: MD5: ffe5d92294b5db62903fcd40a26e11a5
+LicenseConcluded: LicenseRef-FSFUL-a1be7db6af71c8a0c695bd6331de7f1d AND LicenseRef-GPL-2.0-or-later-WITH-Libtool-exception
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Libtool-exception
+LicenseInfoInFile: LicenseRef-FSFUL
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-fsf-free
+LicenseInfoInFile: LicenseRef-libtool-exception-2.0
+FileCopyrightText: <text> Copyright (C) 1992-1996, 1998-2012 Free Software Foundation, Inc.
+Copyright (C) 2012 Free Software Foundation, Inc.
+Copyright (C) 2012 Free Software Foundation, Inc.
+Copyright (C) 2014 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: alsa-lib-1.2.6.1.tar.bz2/alsa-lib-1.2.6.1.tar/alsa-lib-1.2.6.1/config.sub
+SPDXID: SPDXRef-item951308
+FileChecksum: SHA1: acb5dfe011a26adbf9e43e0279166dd412f2a830
+FileChecksum: SHA256: 8dd30cd3972eec02262c317244a9e38ccdf5a0d7fc9eb598edd7e861824cdf8e
+FileChecksum: MD5: 13706dab007cfebc0b59d5fe6ba86a4c
+LicenseConcluded: LicenseRef-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0 AND LicenseRef-warranty-disclaimer
+LicenseInfoInFile: LicenseRef-GPL-3.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-free-unknown
+LicenseInfoInFile: LicenseRef-gpl-3.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception
+FileCopyrightText: <text> Copyright 1992-2018 Free Software Foundation, Inc.
+Copyright 1992-2018 Free Software Foundation, Inc. </text>
+FileAttributionText: <text> This is free software; see the source for copying conditions. There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. </text>
```

Comment:
I think that the text was put in the acknowledgment section by accident, it should be in the "License Text" field